contrib/os-services/roles/prepare/tasks/main.yml

```diff
@@ -0,0 +1,23 @@
+---
+- block:
+  - name: List services
+    service_facts:
+
+  - name: Disable service firewalld
+    systemd:
+      name: firewalld
+      state: stopped
+      enabled: no
+    when:
+      "'firewalld.service' in services"
+
+  - name: Disable service ufw
+    systemd:
+      name: ufw
+      state: stopped
+      enabled: no
+    when:
+      "'ufw.service' in services"
+
+  when:
+  - disable_service_firewall
```

Comment:
Should we add a default value os-services/roles/prepare/defaults/main.yml `disable_service_firewall: true/false`

Reply:
That is a nice point, I will add it.

Reply:
Done